### PR TITLE
Steffi tweaks

### PR DESCRIPTION
**Pull request by steffilazerte**
Pull request to add details regarding image sizes (#166, #171) and info about default tweet text (#170)

### CODE REVIEW

**Review by stefaniebutland — APPROVED** [review]
Thank you Steffi!

File: authortechnical.Rmd [hunk 1]
```diff
@@ -158,22 +158,52 @@ If you're curious about the `description`, `twitterImg`, `twitterAlt` YAML field
...
+
+**Image features**  
+
+- Consider transparent backgrounds as the blog background is not white 
+  (hex logos etc. will look better this way)
+- Keep images < 1 mb
+  - Large images can be compressed/optimized with tools like
+  - [OptiPNG](http://optipng.sourceforge.net/) (Local software, 
+  [also integrates with rmarkdown](https://bookdown.org/yihui/rmarkdown-cookbook/optipng.html))
+  - [minimage](https://github.com/ardata-fr/minimage) (Local software)
```
Comment (maelle) [inline authortechnical.Rmd:189]: I also feel we should explore what magick can help with one day.

**Review by maelle — APPROVED** [review]
Thank you!

**Comment by maelle** [comment]
I had completely forgotten about #170 :joy:

**Pull request merged**